app/utils.py

```diff
@@ -51,3 +53,44 @@ def geocode(address):
     else:
         coords = r.json()['results'][0]['geometry']['location']
         return coords['lat'], coords['lng']
+
+
+def get_current_weather(location):
+    """TODO write docstring"""
```

Comment:
What are the minimum conditions or exceptions for idling incidents? Wondering what conditions are considered relevant data